modules/dataquery/php/module.class.inc

```diff
@@ -3,58 +3,74 @@
  * This serves as a hint to LORIS that this module is a real module.
  * It does nothing but implement the module class in the module's namespace.
  *
- * PHP Version 5
+ * PHP Version 7
```

Comment:
I don't understand why this PR is changing the Module class. Endpoints shouldn't require any changes to the module class, they get auto-delegated to the class with the same name as the page name requested.

Reply:
@driusan I thought it was needed to have the handle ServerRequestInterface inside the module.class.inc otherwise the classes are not delegated to the correct classes in the php/endpoints directory. @xlecours